pkg/scheduler/metrics/job.go

```diff
@@ -37,8 +37,34 @@ var (
 			Help:      "Number of retry counts for one job",
 		}, []string{"job_id"},
 	)
+
+	jobCompletedPhaseCount = promauto.NewCounter(
+		prometheus.CounterOpts{
+			Subsystem: VolcanoNamespace,
+			Name:      "job_completed_phase_count",
+			Help:      "Number of jobs in completed phase",
+		},
+	)
+
+	jobFailedPhaseCount = promauto.NewCounter(
+		prometheus.CounterOpts{
+			Subsystem: VolcanoNamespace,
+			Name:      "job_failed_phase_count",
+			Help:      "Number of jobs in failed phase",
+		},
+	)
 )
 
+// RegisterJobCompleted records the number of jobs in completed phase
+func RegisterJobCompleted() {
+	jobCompletedPhaseCount.Inc()
```

Comment:
Can we add labels? With labels, it will be beneficial for us to query metrics using PQL

Reply:
I've thought about it, but I don't know what to add. Add `JobName`?

Reply:
You can refer to other metrics codes